hr_360_view/wizard/hr_employee_search_wizard.py

```diff
@@ -223,14 +259,18 @@ def search_jobseeker(self):
 
     @api.multi
     def search_jobseeker_authority(self):
+        now_year = str(datetime.now().year)
+        # which gives 2021
+        before_year = str(datetime.now().year - 100)
```

Comment:
before_year is a pretty ambiguous variable name, consider changing to last_century or something more descriptive.